brewtils/models.py

```diff
@@ -802,7 +802,10 @@ def get_instance_by_id(self, id, raise_missing=False):
         return None
 
     def get_instance(self, name):
-        """DEPRECATED: Please use get_instance_by_name instead"""
+        """
+        .. deprecated::3.0
+           Will be removed in 4.0. Use ``get_instance_by_name`` instead
```

Comment:
In cases where the in code deprecation warning didn't specify a version where things will be removed, I just picked 4.0.  If there's a compelling reason why things should linger longer than that, I can change the warning to be more vague.